std_msgs/Header header
sensor_msgs/NavSatFix navsatfix
float64[] enu_true
float64[] enu_gnss_fix
float32[] range_offset
int32[] sats_blocked
int32 num_vis_sat 
int32 num_block_sat
